src/main/java/com/worksap/nlp/elasticsearch/sudachi/ConfigAdapter.kt

```diff
@@ -27,7 +28,7 @@ import kotlin.io.path.exists
 
 class ConfigAdapter(anchor: PathAnchor, settings: Settings, env: Environment) {
   private val basePath = resourcesPath(env, settings)
-  private val fullAnchor = PathAnchor.filesystem(basePath).andThen(anchor)
+  private val fullAnchor = PathAnchor.filesystem(basePath)
```

Comment:
This change means that the dict, char.def, and subplugins **must** be loaded from `config/sudachi`, due to entitlement-policy.yaml.